roles/openshift_hosted/tasks/registry/registry.yml

```diff
@@ -137,7 +137,7 @@
     edits: "{{ openshift_hosted_registry_edits }}"
     force: "{{ True|bool in openshift_hosted_registry_force }}"
 
-- when: openshift_hosted_registry_wait
+- when: openshift_hosted_registry_wait == True
```

Comment:
| bool ?

Reply:
@sdodson The problem is with expansion.  I would end up being, `{{ not (openshift_master_bootstrap_enabled | default(False)) }} | bool` which would still raise an error.

Reply:
@sdodson Ok, I retract my last statement.  I just ran a test and `| bool` was working.  It didn't work when I was trying that approach before.  I'll update the PR. :smile: